### ISSUE TITLE
Wings get hidden when disconecting from world/server

### ISSUE THREAD

**Issue by MolotoveDev** [pau101/Wings#121]
The wings disappear when you disconnect from the world or server, but the effect remains so you can still fly but you look like Superman and that's not cool.

**Comment by codsec** [comment]
did u find a fix or alternative way?

**Comment by MolotoveDev** [comment]
> did u find a fix or alternative way?

Nope, I decided to remove the mod until anyone get me some solution.

**Comment by Whoabear** [comment]
Having same issue with a Server, hopefully there's a fix soon

**Comment by codsec** [comment]
I fix it myself using command blocks (not very good but it works)
